openmetadata-ui/src/main/resources/ui/src/components/JSONSchemaTemplate/BooleanFieldTemplate.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { FieldProps } from '@rjsf/utils';
+import { Col, Row, Switch, Typography } from 'antd';
+import { startCase } from 'lodash';
+import React from 'react';
+
+const BooleanFieldTemplate = (props: FieldProps) => {
+  return (
+    <Row className="m-t-md">
+      <Col span={8}>
+        <Typography>{startCase(props.name)}</Typography>
+      </Col>
+      <Col span={16}>
+        <Switch
+          checked={props.formData}
+          id={props.idSchema.$id}
+          onChange={(value) => props.onChange(value)}
+          onClick={() => props.formContext?.handleFocus?.(props.idSchema.$id)}
+        />
+      </Col>
+    </Row>
```

Comment:
Can't we use Form.Item instead Row/Col?

Reply:
No, here we have to use Row/Col to have a consistent layout and avoid custom CSS.

Reply:
Never mind, yes we can use the `Form.item` with lableCol and labelAlign instead of Row/Col to have a consistent layout.